cvat-ui/src/reducers/notifications-reducer.ts

```diff
@@ -801,6 +805,21 @@ export default function (state = defaultState, action: AnyAction): Notifications
                 },
             };
         }
+        case UserAgreementsActionTypes.GET_USER_AGREEMENTS_FAILED: {
+            return {
+                ...state,
+                errors: {
+                    ...state.errors,
+                    formats: {
+                        ...state.errors.formats,
+                        fetching: {
```

Comment:
probably ``userAgreements`` and ``...state.errors.userAgreements`` are right

Reply:
sure, my bad
fixed